common/src/main/java/com/wynntils/features/user/inventory/ItemHighlightFeature.java

```diff
@@ -71,6 +71,30 @@ public class ItemHighlightFeature extends UserFeature {
     @Config
     public static CustomColor craftedHighlightColor = new CustomColor(0, 138, 138);
 
+    @Config
```

Comment:
Does it really make sense to have highlight enabled per tier? I think a single boolean to enable/disable ingredient highlighting is better.

Reply:
Wynntils has a minimum tier for highlight, I guess I can see this getting used. Not saying we need this 100%, but also not like it is even close to heavy or anything.

Reply:
I'm just thinking it is tricky for the user to disable ingredient highlighting, they need to disable four switches.

Reply:
Ah, I can understand that. I am not sure, I will think about this. I wouldn't do overall toggle PLUS per tier toggles thought, that is for sure.

Reply:
Whatever, it's not really important. I seriously doubt anyone will ever disable this. It's almost one of the raison d'etre of Wynntils...